phpstan.dist.neon

```diff
@@ -3,13 +3,18 @@ includes:
     - dev-tools/vendor/phpstan/phpstan/conf/bleedingEdge.neon
     - dev-tools/vendor/phpstan/phpstan-phpunit/extension.neon
 
+    # Baseline, should only shrink! To regenerate it, just execute:
+    # dev-tools/vendor/bin/phpstan analyse --generate-baseline=dev-tools/phpstan/baseline.php
+    - dev-tools/phpstan/baseline.php
+
 parameters:
     level: 6
     paths:
         - src
         - tests
     excludePaths:
         - tests/Fixtures
+    reportUnmatchedIgnoredErrors: true # Do not allow outdated errors in the baseline
```

Comment:
I think this is already the default behavior.

Reply:
Yes, but personally I like important things to be explicitly configured. I can remove it if project's convention is different 🙂.